docs/stable/store/quickstart.md

```diff
@@ -105,4 +105,144 @@ outputs = model.generate(**inputs)
+
+Thus, for fist-time users, you have to load the model from other backends and then converted it to the ServerlessLLM format.
+
+1. Download the model from HuggingFace and save it in the ServerlessLLM format:
+``` python
+import os
+import shutil
+from typing import Optional
+
+class VllmModelDownloader:
+    def __init__(self):
+        pass
+
+    def download_vllm_model(
+        self,
+        model_name: str,
+        torch_dtype: str,
+        tensor_parallel_size: int = 1,
+        pattern: Optional[str] = None,
+        max_size: Optional[int] = None,
+    ):
+        import gc
+        import shutil
+        from tempfile import TemporaryDirectory
+
+        import torch
+        from huggingface_hub import snapshot_download
+        from vllm import LLM
+        from vllm.config import LoadFormat
+
+        def _run_writer(input_dir, output_dir):
+            # load models from the input directory
+            llm_writer = LLM(
+                model=input_dir,
+                download_dir=input_dir,
+                dtype=torch_dtype,
+                tensor_parallel_size=tensor_parallel_size,
+                num_gpu_blocks_override=1,
+                enforce_eager=True,
+                max_model_len=1,
+            )
+            model_executer = llm_writer.llm_engine.model_executor
+            # save the models in the ServerlessLLM format
+            model_executer.save_serverless_llm_state(
+                path=output_dir, pattern=pattern, max_size=max_size
+            )
+            for file in os.listdir(input_dir):
+                # Copy the metadata files into the output directory
+                if os.path.splitext(file)[1] not in (
+                    ".bin",
+                    ".pt",
+                    ".safetensors",
+                ):
+                    src_path = os.path.join(input_dir, file)
+                    dest_path = os.path.join(output_dir, file)
+                    if os.path.isdir(src_path):
+                        shutil.copytree(src_path, dest_path)
+                    else:
+                        shutil.copy(src_path, output_dir)
+            del model_executer
+            del llm_writer
+            gc.collect()
+            if torch.cuda.is_available():
+                torch.cuda.empty_cache()
+                torch.cuda.synchronize()
+
+        # set the model storage path
+        storage_path = os.getenv("STORAGE_PATH", "./models")
+        model_dir = os.path.join(storage_path, model_name)
+
+        # create the output directory
+        if os.path.exists(model_dir):
+            print(f"Already exists: {model_dir}")
+            return
+        os.makedirs(model_dir, exist_ok=True)
+
+        try:
+            with TemporaryDirectory() as cache_dir:
+                # download model from huggingface
+                input_dir = snapshot_download(model_name, cache_dir=cache_dir)
```

Comment:
This will download all available versions of the model including PyTorch, TensorFlow and many others. Is it possible to only download one format? i.e., PyTorch format.